documentation/modules/rn-2.6.adoc

```diff
@@ -309,6 +316,10 @@ For a complete list of all resolved issues in this release, see the list of link
 
 This release has the following known issues:
 
+.Migrating RHEL and CentOS 7.0 and 7.1 from VMware to {ocp}, the name of the network interfaces changes
+
+When migrating a simple Red Hat Enterprise Linux (RHEL) 7.0 and 7.1, as well as Rocky Linux and CentOS 7.0 and 7.1, virtual machine (VM) from VMware 7 to {ocp} 4.13, the name of the network interfaces changes and the static IP configuration for the VM no longer functions. This issue is caused by RHEL 7.0 and 7.1 still requiring `virtio-transitional`. Workaround: Manually update the guest to RHEL 7.2 or update the VM specification post-migration to use transitional. link:https://issues.redhat.com/browse/MTV-1382[(MTV-1382)]
```

Comment:
Please change
`When migrating a simple Red Hat Enterprise Linux (RHEL) 7.0 and 7.1, as well as Rocky Linux and CentOS 7.0 and 7.1, virtual machine (VM) from VMware 7 to {ocp} 4.13,`

to

`When migrating older Linux distributions, such as CentOS 7.0 and 7.1 virtual machines (VM) from VMware to {ocp},`